src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private static final Random RANDOM = new Random();
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
```

Comment:
class Ball does not need these. 
it only represents the Ball and gives ability to create a new one. 
It should not creates in random way every time (for this we have lottery).
```suggestion
```